library/globals.inc.php

```diff
@@ -3092,5 +3092,40 @@ function gblTimeZones()
             xl('Select to print patient address labels from popups in left nav screen.')
         ),
 
+        'env_x_width' => array(
+            xl('Envelope Width in mm'),
+            'num',                           // data type
+            '104.775',                              // default = false
+            xl('In Portrait mode, determines the width of the envelope along the x-axis in mm')
+        ),
+
+        'env_y_height' => array(
+            xl('Envelope Height in mm'),
+            'num',                           // data type
+            '241.3',                              // default = false
+            xl('In Portrait mode, determines the height of the envelope along the y-axis in mm')
+        ),
+
+        'env_font_size' => array(
+            xl('Font Size in Pt'),
+            'num',                           // data type
+            '14',                              // default = false
+            xl('Sets the font of the address text on the envelope in mm')
+        ),
+
+        'env_x_dist' => array(
+            xl('Envelope x-axis starting pt'),
+            'num',                           // data type
+            '65',                              // default = false
+            xl('Distance from the \'top\' of the envelope in mm')
+        ),
+
+        'env_y_dist' => array(
+            xl('Envelope y-axis starting pt'),
+            'num',                           // data type
+            '220',                              // default = false
+            xl(' Distance from the right most edge of the envelope in portrait position in mm')
+        ),
+
```

Comment:
If you look at user settings, developer gets to specify the 'default' which is stored for user '0'.  Then users override it and it should stick until they change it.  They do that all the time with their other print jobs.

My comment was more towards the general approach of using global settings for something that is applicable to 1 or 2 screens.  In age of apps, users have grown to expect the 'Settings' button somewhere that would immediately affect the current screen.

Sorry for choosing a wrong place to insert this discussion.